aspnet/tutorials/first-mvc-app/adding-model.rst

```diff
@@ -34,72 +31,64 @@ In the **Add Scaffold** dialog, tap **MVC Controller with views, using Entity Fr
-  - The ``ApplicationDbContextModelSnapshot`` class creates the EF entities used to access the Identity database. We'll talk more about EF entities later in the tutorial.
+  A database operation failed while processing the request.
+  SqlException: Invalid object name 'Movie'. 
+  There are pending model changes for ApplicationDbContext
+  Scaffold a new migration for these changes and apply them to the database from the command line:
+  > dotnet ef migrations add [migration name] 
+  > dotnet ef database update 
 
-Visual Studio automatically created the CRUD (create, read, update, and delete) action methods and views for you (the automatic creation of CRUD action methods and views is known as *scaffolding*). You'll soon have a fully functional web application that lets you create, list, edit, and delete movie entries.
+That's a great error message, we'll follow those instructions to get the database ready for our Movie app.
 
 Use data migrations to create the database
 --------------------------------------------
 
 - Open a command prompt in the project directory (MvcMovie/src/MvcMovie). Follow these instructions for a quick way to open a folder in the project directory.
 
-  - Open a file in the root of the project (for this example, use *Startup.cs*.)
-  - Right click on *Startup.cs*  **> Open Containing Folder**.
+ 	- Open a file in the root of the project (for this example, use *Startup.cs*.)
+ 	- Right click on *Startup.cs*  **> Open Containing Folder**.	
 
-  .. image:: adding-model/_static/quick.png
+	.. image:: adding-model/_static/quick.png
 
-  - Shift + right click a folder > **Open command window here**
+	- Shift + right click a folder > **Open command window here**
+	
+	.. image:: adding-model/_static/folder.png
 
-  .. image:: adding-model/_static/folder.png
-
-  - Run ``cd ..`` to move back up to the project directory
+ 	- Run ``cd ..`` to move back up to the project directory
 
 - Run the following commands in the command prompt:
 
 .. code-block:: PHP
 
-  dotnet restore
-  dotnet ef migrations add Initial
-  dotnet ef database update
-
-.. image:: adding-model/_static/cmd.png
-
-- The ``ef`` command is specified in the *project.json* file:
-
-.. literalinclude:: start-mvc/sample/src/MvcMovie/project.json
-  :language: JSON
-  :lines: 30-34
-  :dedent: 0
-  :linenos:
-  :emphasize-lines: 3
-
-- ``dotnet ef migrations add Initial`` Creates a class named ``Initial``
-
-.. code-block:: c#
+	dotnet ef migrations add Initial
+	dotnet ef database update	
 
-  public partial class Initial : Migration
+- ``dotnet`` (.NET Core) is a cross-platform implementation of .NET. You can read about it `here <http://dotnet.github.io/about/>`__.
```

Comment:
@danroth27 
http://go.microsoft.com/fwlink/?LinkId=798644
I'll update it when we get a better target